Base/home/anon/WindowManager.ini

```diff
@@ -1,6 +1,6 @@
 [Screen]
 Width=1080
-Height=768
+Height=1080
```

Comment:
Let's leave the default screen size at 1024x768 :)

Reply:
Oh DARN. That wasn't supposed to make it through.

Reply:
Fixed.